brat/Brat/Syntax/Common.hs

```diff
@@ -164,6 +174,8 @@ deriving instance Eq io => Eq (CType' io)
 instance Semigroup (CType' (PortName, ty)) where
   (ss :-> ts) <> (us :-> vs) = (ss <> us) :-> (ts <> vs)
 
+type UserNameMap = M.Map End String
```

Comment:
How about calling this `NamesForMetas` or `NamesForEnds`? (Is "Metas" a new term in this PR and if so do we want to suddenly introduce it like so, or is it a term we've been using in the code previously?